crates/sui-open-rpc/spec/openrpc.json

```diff
@@ -317,7 +317,7 @@
                 "txSignature": "AACunVNQ+43nTvpMty4ILN+tTvYAaUsYn3Spu1eZu7SZVTPs5tkYxPF8h1o6iFDgDgq/rrua+hMNxMkGYVhXtw+NZVNaIOpz/rPnTIA7TCFEizybXsMse2pIqWIapAFigQ==",
                 "authSignInfo": {
                   "epoch": 0,
-                  "signature": "AaCQESRLSxMntCxHAXuRk66nmlID2DZn3gU64vkZ7iHSjE2Nbhfr21Ovri7AMdAl4Q==",
+                  "signature": "wAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```

Comment:
this is inf, right? i'm assuming there is a test that generates these examples, can we update it to use something else (e.g., just the generator)?

Reply:
will follow up a PR, currently generator() is on G1Element which requires some massage into a BLS12381Signature. will update fastcrypto first